client/python/armada_client/client.py

```diff
@@ -146,29 +146,67 @@ def reprioritize_jobs(
         response = self.submit_stub.ReprioritizeJobs(request)
         return response
 
-    def create_queue(self, name: str, **queue_params) -> empty_pb2.Empty:
+    def create_queue(
+        self,
+        name: str,
+        priority_factor: Optional[int],
```

Comment:
This is actually a double in the API definition.  I think double->float is fine?  
```suggestion
        priority_factor: Optional[float],
```

```
    double priority = 1;
```